src/model/Computation.js

```diff
@@ -65,7 +67,7 @@ export default class Computation extends Model {
 		this.boundsSensitive = true;
 		this.dirty = true;
 
-		// TODO: computations don't shuffle, but this is a bit hackish
+		// TODO: is there a less hackish way to do this?
 		this.shuffle = undefined;
```

Comment:
```js
this.shuffle = void 0;
```
:trollface:

If this eventually becomes a function, a noop/empty function perhaps? That way, one can safely assume it's a callable all the time regardless if it does something or nothing.